scripts/etc/init.d/rcS

```diff
@@ -409,6 +413,15 @@ case "$rootfstype" in
     ;;
 esac
 
+# check if we need wpasupplicant package
```

Comment:
And we assume that this is a wireless interface and therefore the `wpasupplicant` package is needed.